components/04-views/dashboard/product-reviews/product-reviews.hbs

```diff
@@ -0,0 +1,17 @@
+{{ render '@header' }}
+
+<div class="dashboard container">
+    <div class="grid">
+        <div class="grid__column grid__column--1-4 dashboard__sidebar">
+            {{ render '@dashboard-nav' }}
+        </div>
+
+        <div class="grid__column grid__column--3-4 dashboard__content">
+            {{> '@dashboard-table' reviews }}
+
+            {{> '@dashboard-actions' button }}
+        </div>
+    </div>
+</div>
+
+{{ render '@footer' }}
```

Comment:
missing rating stars and view of single review (like in magento default)

Reply:
added, added

Reply:
👍